packages/web/src/dev/CopyToClipboard.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import CopyIcon from 'src/icons/CopyIcon'
+import { colors } from 'src/styles'
+import { copyToClipboad } from 'src/utils/utils'
+
+interface Props {
+  content: string
+}
+
+interface State {
+  copied: boolean
+  hover: boolean
+}
+
+class CopyToClipboard extends React.PureComponent<Props, State> {
+  state = {
+    copied: false,
+    hover: false,
+  }
+
+  copy(event) {
+    event.preventDefault()
+    event.stopPropagation()
+    if (!this.state.copied) {
+      copyToClipboad(this.props.content)
+      this.setState({ copied: true })
+      setTimeout(() => this.setState({ copied: false }), 2000)
+    }
+  }
+
+  // Simulating css hover
+  // hover(hover: boolean) {
+  //   this.setState({hover})
+  // }
+
+  render() {
+    const { copied, hover } = this.state
+    const copy = this.copy.bind(this)
+    return (
+      <View
+        style={[styles.container]}
+        onClick={copy}
+        /*onMouseOver={this.hover.bind(this, true)}*/
```

Comment:
is this in progress?

Reply:
Will reuse with Hoverable component, thanks!